src/main/java/net/glowstone/block/entity/state/GlowDispenser.java

```diff
@@ -92,12 +98,24 @@ public static void register() {
                     InventoryUtil.copyPotionDataToTippedArrow(tippedArrow, itemStack);
                     return tippedArrow;
                 })));
-        registry.putBehavior(Material.FIREWORK_CHARGE, new ProjectileDispenseBehavior(location -> {
```

Comment:
Was this removal intentional?

Reply:
Yes: should have been FIREBALL.

Reply:
Do fireworks no longer get let off by dispensers? Or is `firework_charge` something else?

Reply:
I'm pretty sure FIREWORK_CHARGE is a firework *star*. The dispense behavior for fireworks will not be a ProjectileDispenseBehavior, because GlowFirework doesn't implement Projectile.

Reply:
Ah okay, I thought the firework entities were considered projectiles.

Reply:
Adding a TODO in the next commit.